cookbook/integrations/aws/sagemaker_pytorch/requirements.txt

```diff
@@ -178,11 +178,13 @@ stringcase==1.2.0
     # via dataclasses-json
 tensorboardx==2.1
     # via -r requirements.in
-torch==1.8.0
+    # torch==1.8.1+cpu doesn't work currently.Hence using the wheel link directly.
+https://download.pytorch.org/whl/cpu/torch-1.8.1%2Bcpu-cp38-cp38-linux_x86_64.whl
```

Comment:
Did you manually update this file? It'll get overwritten when we run `make requirements`. You can do the same change to requirements.in

Reply:
Thanks . Missed the auotgenerate at the top of the file. Moved this to requirements.in file now